scripts/ci/verify_linter.py

```diff
@@ -0,0 +1,183 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import json
+from subprocess import check_output, check_call
+from pkg_resources import parse_version
+
+
+def separator_line():
+    print('-' * 100)
+
+
+class ModifiedFilesNotAllowedError(Exception):
+    """
+    Exception raise for the scenario that modified files is conflict against publish requirement.
+    Scenario 1: if modified files contain only src/index.json, don't raise
+    Scenario 2: if modified files contain not only extension code but also src/index.json, raise.
+    Scenario 3: if modified files don't contain src/index.json, don't raise.
```

Comment:
You may want to consider the edge case of changes to delete an extension, such as extension moved to module.

Reply:
Yes, it's an edge case. Working on it.

Reply:
It's of low priority. You can improve it in later sprint since such cases are rare and even if it occurs, we can force merge PRs with CI failure. No big risk for just deleting the exension.

Reply:
Fixed. 
I added --diff-filer=ACMRT to git diff both in this pipeline and the internal release pipeline to exclude deleted files.